REFERENCE.md

```diff
@@ -42,849 +43,1033 @@ repositories.
 
 ## Classes
 
-### `redis`
+### <a name="redis"></a>`redis`
```

Comment:
Could you regenerate `REFERENCE.md` prior to the changes in a separate commit to make reviewing easier?

Reply:
@ekohl I did the split to two commits